Directory.Build.props

```diff
@@ -28,9 +28,13 @@
         <IncludeSymbols>true</IncludeSymbols>
         <SymbolPackageFormat>snupkg</SymbolPackageFormat>
         <EnableTrimAnalyzer>true</EnableTrimAnalyzer>
+
+        <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
```

Comment:
This should ideally at some point be true (to give a TODO list)

Reply:
Yeah exactly. My approach is to try and fix warnings incrementally. Once that's reduced to a manageable size, we can flip this switch and fix the remaining ones and or ignore the ones we don't care about (if any).